internal/home/service.go

```diff
@@ -231,20 +231,32 @@ func handleServiceInstallCommand(s service.Service) {
 			log.Fatal(err)
 		}
 	}
-
-	// Start automatically after install
-	err = svcAction(s, "start")
-	if err != nil {
-		log.Fatalf("Failed to start the service: %s", err)
-	}
-	log.Printf("Service has been started")
-
+	
+	if runtime.GOOS != "freebsd" {
+	    // Start automatically after install
+	    err = svcAction(s, "start")
+	    if err != nil {
+	    	log.Fatalf("Failed to start the service: %s", err)
+    	}
+    	log.Printf("Service has been started")
```

Comment:
It looks like the code hasn't been `gofumpt`ed?  Did `make go-check` exit with success?`